Marlin/Configuration.h

```diff
@@ -660,6 +660,12 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
 //#define VIKI2
 //#define miniVIKI
 
+// This is a new controller currently under development.  A link to more information will be provided as it
+// becomes available.
+//
+// ==> REMEMBER TO INSTALL U8glib to your ARDUINO library folder: http://code.google.com/p/u8glib/wiki/u8glib
+//#define ELB_FULL_GRAPHIC_CONTROLLER
+
```

Comment:
Sorry, I forget that line notes aren't always displayed with the lines they refer to.